seahash.pyi

```diff
@@ -1,3 +1,5 @@
+from collections.abc import Buffer
```

Comment:
Issue: `Buffer` only exists in 3.12, so this won't work on older versions. I think just using `memoryview` is fine here.

Reply:
Good catch!